packages/remix/src/utils/utils.ts

```diff
@@ -73,3 +79,10 @@ export const getEnvVariable = (name: string, context: AppLoadContext | undefined
 
   return '';
 };
+
+export const getSpaMode = (): boolean => {
+  if (typeof window !== 'undefined' && typeof window.__remixContext?.isSpaMode !== 'undefined') {
+    return window.__remixContext.isSpaMode;
+  }
```

Comment:
@anagstef could you add a link where `window.__remixContext.isSpaMode` is mentioned in their docs ? I couldn't find it.

Reply:
@panteliselef It's not mentioned because it is kinda internal. My initial proposal was to introduce a `spaMode: true` prop option, but discussed with @octoper and this solution has better DX.